src/microbe_stage/MicrobeStage.cs

```diff
@@ -228,6 +228,10 @@ public void MoveToEditor()
 
         var editor = (MicrobeEditor)scene.Instance();
 
+        var playerSpecies = GameWorld.PlayerSpecies;
+        GameWorld.AlterSpeciesPopulation(
+            playerSpecies, Constants.PLAYER_REPRODUCE_POPULATION_GAIN, "player reproduced", true);
+
```

Comment:
Just to be safe, I'd like these lines to be before the editor is created. As it's the editor's job to apply this, so some crazy changed there could break this code. Also this shouldn't use the instant effect. As the comment on that says, it is *only* for the player dying.